test/fitnesse/components/PluginsClassLoaderTest.java

```diff
@@ -16,66 +16,29 @@
 
   @Test
   public void whenPluginsDirectoryDoesNotExist() throws Exception {
-    PluginsClassLoader pluginsClassLoader = new PluginsClassLoader("nonExistingRootDirectory");
-    pluginsClassLoader.addPluginsToClassLoader();
+    new PluginsClassLoader().loadPlugins("nonExistingRootDirectory");
+
     assertTrue("didn't cause exception", true);
   }
 
   @Test
   public void addPluginsToClassLoader() throws Exception {
     String[] dynamicClasses = new String[]{"fitnesse.testing.PluginX", "fitnesse.testing.PluginY"};
     //todo This fails because some other test probably loads plugin path    assertLoadingClassCausesException(dynamicClasses);
```

Comment:
This todo line can be removed, as the `assertLoadingClassCausesException` method no longer exists

Reply:
Fixed.